README.md

```diff
@@ -1 +1,122 @@
+[![GitHub release](https://img.shields.io/github/release/ramendr/ramen/all.svg)](https://github.com/ramendr/ramen/releases)
+[![Docker Pulls](https://img.shields.io/docker/pulls/ramen/ceph)](https://hub.docker.com/u/ramen)
+[![Go Report Card](https://goreportcard.com/badge/github.com/ramendr/ramen)](https://goreportcard.com/report/github.com/ramendr/ramen)
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/1599/badge)](https://bestpractices.coreinfrastructure.org/projects/1599)
+[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Framendr%2Framen.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Framendr%2Framen?ref=badge_shield)
+[![Slack](https://slack.ramen.io/badge.svg)](https://slack.ramen.io)
+[![Twitter Follow](https://img.shields.io/twitter/follow/ramen_io.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=ramen_io&user_id=788180534543339520)
+
+-->
+
+# What is Ramen?
+
+Ramen is an open source **disaster-recovery service** for stateful applications running on
+OpenShift Container Storage (OCS).  Ramen provides a simplified interface to automate the life-cylce of an 
+application during disastrous events; these include failover of an application to a peer cluster and failback
+of the application back to the home cluster. Ramen provides cloud-native interfaces to enable a kubernetes
+cluster administrator to setup and manage disaster recovery of stateful applications using OCS across a set of
+independent kubernetes clusters. Ramen replicates k8s metadata of a group of PVs/PVCs of a given application
+across peer clusters and manages OCS’ replication of data in those PVs across the same peer clusters.
+
+Ramen integrates deeply into cloud native environments leveraging extension points to provide a seamless
+experience for scheduling replication of PVs, lifecycle management, resource management, security, monitoring,
+and user experience.
+
+# Ramen Usecase
+Consider a Kubernetes cluster that is running a stateful application; let us call this cluster the
+home cluster for that application.  To prepare for a disastrous event affecting the home cluster region, one
+needs to provision one or more peer clusters in alternate regions (away from the home cluster region) and
+replicate the setup on home cluster to the peer cluster, which requires the following:
+A. Replicate the application deployment configuration in the home cluster to the peer cluster.
+   running in the home cluster to the peer clusters.  Ramen relies on Red Hat ACM to meet this requirement.
+B. Replicate the data of all the PVs (PersistentVolumes) of the application in the home cluster to
+   the one of the peer clusters.  Ramen relies on Red Hat OCS to meet this requirement.
+C. Replicate the etcd metadata of all the PVs of the application in the home cluster to one of
```

Comment:
I would remove etcd from here.  you may want to say something like K8s PV object resources or something like that.